### PR TITLE
Fix issue bug 63

### PR DESCRIPTION
**Pull request by loay**
Just defining the baseurl so the app runs fine

### CODE REVIEW

**Comment by superkhau** [comment]
Can you move it up a few lines like how we do it here: https://github.com/strongloop/loopback-getting-started/blob/master/server/server.js#L10-L14

File: server/server.js [hunk 1]
```diff
@@ -177,6 +177,7 @@ app.start = function() {
     console.log('Web server listening at: %s', app.get('url'));
     if (app.get('loopback-component-explorer')) {
       var explorerPath = app.get('loopback-component-explorer').mountPath;
+      var baseUrl = app.get('url');
```
Comment (superkhau) [inline server/server.js]: Move this up a few lines and change the code to match https://github.com/strongloop/loopback-example-passport/pull/64

**Comment by loay** [comment]
changes done

**Comment by superkhau** [comment]
LGTM

**Pull request merged**